### PR TITLE
Add CC-BY image and ISSN to footer (#424)

### PR DESCRIPTION
**Pull request by thatbudakguy**

### CODE REVIEW

File: templates/snippets/footer.html [hunk 1]
```diff
@@ -21,17 +21,23 @@
...
+                <a href="https://twitter.com/ProsodyArchive">
+                    <img src="{% static 'img/icons/twitter.svg' %}" alt="ProsodyArchive on Twitter"/>
+                </a>
+                <a href="https://www.instagram.com/prosodyarchive/">
+                    <img src="{% static 'img/icons/instagram.svg' %}" alt="ProsodyArchive on  Instagram"/>
+                </a>
+            </div>
+        </div>
         <div class="contact ui inverted horizontal link list">
             <span class="item">©2018 Trustees of Princeton University</span>
```
Comment (rlskoeser) [inline templates/snippets/footer.html:42]: while we're at it, should we update the date here? or always use the current year? I think you did that on another project

File: templates/snippets/footer.html [hunk 2]
```diff
@@ -21,17 +21,23 @@
...
-            <a href="https://twitter.com/ProsodyArchive">
-                <img src="{% static 'img/icons/twitter.svg' %}" alt="ProsodyArchive on Twitter"/>
-            </a>
-            <a href="https://www.instagram.com/prosodyarchive/">
-                <img src="{% static 'img/icons/instagram.svg' %}" alt="ProsodyArchive on  Instagram"/>
-            </a>
-        </section>
+        <div class="media">
+            <div class="left">
+                <a class="license" href="https://creativecommons.org/licenses/by/4.0/"><img src="{% static 'img/logos/cc-by.svg' %}"></a>
```
Comment (rlskoeser) [inline templates/snippets/footer.html:26]: shouldn't we have some alt text for a screen-readable label saying what the license is? (creative commons attribution?)

**Review by rlskoeser — APPROVED** [review]
Looks good to me! Can we update year while we're at it?

File: templates/snippets/footer.html (hunk 2 above)
Comment (thatbudakguy) [reply templates/snippets/footer.html:26]: oh, duh, thank you! Will add

File: templates/snippets/footer.html (hunk 1 above)
Comment (thatbudakguy) [reply templates/snippets/footer.html:42]: yeah, I'll make it always use the current year

**Pull request merged**